pkg/gui/presentation/branches.go

```diff
@@ -177,30 +152,38 @@ func BranchStatus(
 ) string {
 	itemOperationStr := ItemOperationToString(itemOperation, tr)
 	if itemOperationStr != "" {
-		return itemOperationStr + " " + utils.Loader(now, userConfig.Gui.Spinner)
-	}
-
-	if !branch.IsTrackingRemote() {
-		return ""
-	}
-
-	if branch.UpstreamGone {
-		return tr.UpstreamGone
-	}
-
-	if branch.MatchesUpstream() {
-		return "✓"
-	}
-	if branch.RemoteBranchNotStoredLocally() {
-		return "?"
+		return style.FgCyan.Sprintf("%s %s", itemOperationStr, utils.Loader(now, userConfig.Gui.Spinner))
 	}
 
 	result := ""
-	if branch.IsAheadForPull() {
-		result = fmt.Sprintf("↑%s", branch.AheadForPull)
-	}
-	if branch.IsBehindForPull() {
-		result = fmt.Sprintf("%s↓%s", result, branch.BehindForPull)
+	if branch.IsTrackingRemote() {
+		if branch.UpstreamGone {
+			result = style.FgRed.Sprint(tr.UpstreamGone)
+		} else if branch.MatchesUpstream() {
+			result = style.FgGreen.Sprint("✓")
+		} else if branch.RemoteBranchNotStoredLocally() {
+			result = style.FgMagenta.Sprint("?")
+		} else if branch.IsBehindForPull() && branch.IsAheadForPull() {
+			result = style.FgYellow.Sprintf("↓%s↑%s", branch.BehindForPull, branch.AheadForPull)
```

Comment:
I'm against this change for a few reasons:
* the original order is more intuitive for me (if only via familiarity)
* there is no standard ordering in place across git clients so this doesn't make us any more standard (which would have been a good reason otherwise)
* Lazygit users will have become accustomed to the existing ordering and may find it jarring if it's changed.

That third point is my main gripe: I suggest that we should create an issue for this specifically and see what other people think: if a wide majority share your perspective we can change it. But I'd leave it out of this PR

Reply:
Let me ask you: did you even notice the change yourself before you read about it in the PR? Please be honest 😄. The change was in the prototype from the very beginning.

I'd like to avoid having to do the extra work of backing out the change again and making a separate issue and PR; I personally feel the change is so insignificant that it's not worth it.

BTW, I looked again at all the git clients I have on my machine: VS Code, Fork, and Sublime Merge. They all show `↓` first. The only counter example I could find is git itself in its "git status" and "git branch -v" output, but that one doesn't use arrows.

Reply:
> Let me ask you: did you even notice the change yourself before you read about it in the PR? Please be honest 😄. The change was in the prototype from the very beginning.

That is a very good question haha. I definitely noticed the commit on the branch before I saw the change in this pull request, and I definitely noticed that commit before I noticed the change in the UI itself, but I don't have the counterfactual of how I would respond otherwise.

Alas, happy to just YOLO this one and see if anybody (including myself) find it jarring upon release